.github/workflows/sensitive_files.txt

```diff
@@ -0,0 +1,15 @@
+.github/
+env.example
+.husky/
+scripts/
+package.json
+tsconfig.json
+.gitignore
```

Comment:
The list here must exactly match what was in the original `paths:` key. You have changed the functionality